### PR TITLE
Update iOS doc to match Android's.

### PR DESCRIPTION
**Pull request by avi-kik**

### CODE REVIEW

File: docs/ios.md [hunk 1]
```diff
@@ -29,34 +29,42 @@ git clone --recursive https://github.com/kinfoundation/kin-core-ios
...
+```swift
+kinClient.deleteAccount(at: 0)
+```
 
-### KinAccount
+## Onboarding
 
 Before an account can be used on the configured network, it must be funded with the native network currency.
-This step must be performed by a service, and is outside the scope of this SDK.
+This step must be performed by a service, such as the <a href="https://github.com/kinecosystem/kin-core-docs/blob/7f0714ba6f317401de7c5878f93ca574e5edec4c/docs/fee-faucet.md">Native Asset Faucet Service</a>.
```
Comment (yosriz) [inline docs/ios.md]: use markdown relative links:
[Native Asset Faucet Service](fee-faucet.md)

File: docs/ios.md [hunk 2]
```diff
@@ -29,34 +29,42 @@ git clone --recursive https://github.com/kinfoundation/kin-core-ios
...
+
+```swift
+kinClient.deleteAccount(at: 0)
+```
 
-### KinAccount
+## Onboarding
 
 Before an account can be used on the configured network, it must be funded with the native network currency.
-This step must be performed by a service, and is outside the scope of this SDK.
```
Comment (yosriz) [inline docs/ios.md]: Can you add a word or 2 for the onboarding guide at [Onboarding](onboarding.md), also provide a complete example for the onboarding (against kin faucet and native faucet like in sample)

**Comment by yosriz** [comment]
@avi-kik thanks, please see my comments

File: docs/ios.md [hunk 3]
```diff
@@ -70,64 +78,115 @@ account.activate(completion: { txHash, error in
...
+                        memo: "payment for donuts",
+                        completion: { txHash, errors
+                          if let txHash = txHash {
+                            // payment succeeded
+                          }
+                        })
+```
 
----
+The `memo` field can contain a utf-8 string up to 28 bytes in length.  A typical usage is to include an order# that a service can use to verify payment.
```
Comment (yosriz) [inline docs/ios.md]: can you add this to kin-core-ios docs also? (code documentation)

Comment (avi-kik) [reply docs/ios.md]: This is done.

File: docs/ios.md (hunk 2 above)
Comment (avi-kik) [reply docs/ios.md]: I'll add the same text you have in the Android doc -- a pointer to the sample app.

**Pull request merged**